flojoy/data_container.py

```diff
@@ -213,76 +211,131 @@ def validate(self):
 
 
 class OrderedPair(DataContainer):
-    def __init__(self, x: DCNpArrayType, y: DCNpArrayType):
+    x: DCNpArrayType
```

Comment:
Why add these as class attributes? Is it for the type checker?

Reply:
Yes for type hint